lib/tty/file/create_file.rb

```diff
@@ -59,6 +63,11 @@ def convert_encoded_path(filename)
       #
       # @api private
       def detect_collision
+        if directory?
+          notify(:error, :red)
+          return
+        end
```

Comment:
I don't believe this is necessary if we allow copying a file to a directory destination. This should never be the case unless someone tries to copy a directory to directory, I guess we could test for that case. This would require another test case.